topology/probes/k8s/cache.go

```diff
@@ -37,40 +37,25 @@ import (
 	"k8s.io/client-go/tools/cache"
 )
 
-// ResourceHandler is used to map Kubernetes resources to objets in the graph
-type ResourceHandler interface {
-	Map(obj interface{}) (graph.Identifier, graph.Metadata)
-	Dump(obj interface{}) string
-	IsTopLevel() bool
+type k8sHandler interface {
```

Comment:
I think we should stop renaming things

Reply:
I have split the original functionality of ResouceHandler into two:
- k8sHandler - handling operations triggered by k8s client on changes a k8s.resources  
- ResourceHandler - to link k8s.resources to graph.nodes
This split enables handling both "standard" resources (such as pod) as well as "special" resources (such as container)

```
type k8sHandler interface {
        OnAdd(obj interface{})
        OnUpdate(oldObj, newObj interface{})
        OnDelete(obj interface{})
}

type ResourceHandler interface {
        Map(obj interface{}) (graph.Identifier, graph.Metadata)
        Dump(obj interface{}) string
}
```